hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -56,6 +56,29 @@ export const getTeamOrders = createAsyncThunk<
     }
 });
 
+export const updateOrder = createAsyncThunk<
+    OrderInTable,
+    number,
+    { state: RootState; rejectValue: RejectValue; dispatch: AppDispatch }
+>(`${orderReducerName}/updateOrder`, async (order_id, { rejectWithValue }) => {
+    try {
+        const data = {
+            status: "Cancelled",
+            request: "string",
+        };
+        const response = await patch<APIListResponse<Order>>(
+            `/api/hardware/orders/${order_id}`,
+            data
```

Comment:
since the request json has only one property, I think you can put it directly in here.